cmake/features.cmake

```diff
@@ -49,6 +49,8 @@ ov_dependent_option (ENABLE_GPU_DEBUG_CAPS "enable GPU debug capabilities at run
 ov_dependent_option (ENABLE_CPU_DEBUG_CAPS "enable CPU debug capabilities at runtime" ON "ENABLE_DEBUG_CAPS;ENABLE_INTEL_CPU" OFF)
 ov_dependent_option (ENABLE_SNIPPETS_DEBUG_CAPS "enable Snippets debug capabilities at runtime" ON "ENABLE_DEBUG_CAPS" OFF)
 
+ov_dependent_option (ENABLE_SNIPPETS_LIBXSMM_TPP "allow Snippets to use LIBXSMM Tensor Processing Primitives" OFF "ENABLE_INTEL_CPU" OFF)
```

Comment:
1) This was intentionally named like this to align with `ENABLE_SNIPPETS_DEBUG_CAPS`. So I don't think we need to add the `CPU` suffix here.
2) It is a dependent option, it means you can't set `ENABLE_SNIPPETS_LIBXSMM_TPP ` to true if `ENABLE_INTEL_CPU` was not set to true.
3) User can build libxsmm with ov on non x86 platforms if he/she wish to do so. Why not? It's just that nothing would happen, since tokenization is not supported on non-x86 yet.

Reply:
Ok, you convinced me 😁 
I changed the line to `ov_dependent_option (ENABLE_SNIPPETS_LIBXSMM_TPP "allow Snippets to use LIBXSMM Tensor Processing Primitives" OFF "ENABLE_INTEL_CPU AND X86_64" OFF)`